game/resources.cpp

```diff
@@ -709,31 +770,32 @@ Dx8::PatternList Resources::loadDxMusic(std::string_view name) {
   return inst->implLoadDxMusic(name);
   }
 
-const ProtoMesh* Resources::decalMesh(const ZenLoad::zCVobData& vob) {
+const ProtoMesh* Resources::decalMesh(const phoenix::vob& vob) {
   std::lock_guard<std::recursive_mutex> g(inst->sync);
   return inst->implDecalMesh(vob);
   }
 
-ZenLoad::oCWorldData Resources::loadVobBundle(std::string_view name) {
+std::vector<std::unique_ptr<phoenix::vob>>& Resources::loadVobBundle(std::string_view name) {
```

Comment:
Yes, if the called saves the returned reference. Since this function is mostly used to load small VOb trees I can change it to copy instead, like it was doing before

Reply:
Please do copy)

Reply:
@Try Copying proved impractical. I've resorted to doing it the same way all other cached values are used; that is: use a `std::unique_ptr` and return a pointer from the function. To ensure this actually functions as intended I had to make the `phoenix::vob` parameter to `Vob::load` const.